docs/slurm-troubleshooting.md

```diff
@@ -90,6 +91,23 @@ The solution here is to [request more of the specified quota](#gcp-quotas),
 
 [partition-machine-type]: community/modules/compute/schedmd-slurm-gcp-v6-nodeset/README.md#input_machine_type
 
+#### Insufficient Capacity
+
+It may be that the zone the partition is deployed in has no remaining capacity to create the
+compute nodes required to run your submitted job.
+
+You can confirm this by SSHing into the `controller` VM and checking the
+`resume.log` file:
+
+```shell
+$ cat /var/log/slurm/resume.log
```

Comment:
I believe sudo is required to read this log, correct? If so lets include it here so people know that is expected.

Reply:
Good point - updated.